README.md

```diff
@@ -15,18 +15,16 @@ Avoid using too heavy models, because the computation time required to load them
 Then, unzip the model in your app folder. If you just need to use the iOS version, put the model folder wherever you want, and import it as described below. If you need both iOS and Android to work, you can avoid to copy the model twice for both projects by importing the model from the Android assets folder in XCode. Just do as follow:
 
 ### Android
-In Android Studio, open the project manager, right-click on your project folder and New > Folder > Assets folder.
-![Android Studio assets folder creation](https://raw.githubusercontent.com/riderodd/react-native-vosk/main/docs/android_studio_assets_folder_creation.png)
+In Android Studio, open the project manager, right-click on your project folder and go to `New` > `Folder` > `Assets folder`.
+
+Then put the model folder inside the assets folder created. In your file tree it should be located in `android\app\src\main\assets`. So, if you downloaded the french model named `model-fr-fr`, you should access the model by going to `android\app\src\main\assets\model-fr-fr`. In Android studio, your project structure should be like that:
 
-Then put the model folder inside the assets folder created. In your file tree it should be located in android\app\src\main\assets. So, if you downloaded the french model named model-fr-fr, you should access the model by going to android\app\src\main\assets\model-fr-fr. In Android studio, your project structure should be like that:
 ![Android Studio final project structure](https://raw.githubusercontent.com/riderodd/react-native-vosk/main/docs/android_studio_project_structure.png)
 
 You can import as many models as you want.
 
 ### iOS
-In XCode, right-click on your project folder, and click on "Add files to [your project name]".
-
-![XCode add files to project](https://raw.githubusercontent.com/riderodd/react-native-vosk/main/docs/xcode_add_files_to_folder.png)
```

Comment:
Why removing the picture ?

Reply:
Also I'd like to have a js based loader, would be easier installation, smth like:

```js
import modelEN from "models/model-en"

vosk.load(modelEN)
```

But I don't know if that even possible, wdy think ?

Reply:
I added back images for now !

Reply:
> Also I'd like to have a js based loader, would be easier to install and maintain, like:
> 
> ```js
> import modelEN from "models/model-en"
> 
> vosk.load(modelEN)
> ```
> 
> But I don't know if that even possible, wdy think ?

Unfortunately no, it's not. At least when I first researched how to include a file in the final bundle, maybe we can find a way now, but some months ago I didn't figured it out.